Trakt/ScheduledTasks/SyncFromTraktTask.cs

```diff
@@ -336,42 +336,41 @@ private async Task SyncTraktDataForUser(Jellyfin.Data.Entities.User user, double
                         tLastReset = resetValue;
                     }
 
-                    // Fallback procedure to find match by using episode history
-                    if (matchedWatchedSeason == null && matchedWatchedEpisodeHistory != null)
+                    // Check is match is found via history, fallback to match by season and number
+                    if (matchedWatchedEpisodeHistory != null && matchedWatchedEpisodeHistory.Any())
```

Comment:
```suggestion
                    if (matchedWatchedEpisodeHistory is not null && matchedWatchedEpisodeHistory.Any())
```

Reply:
Sure about this? It uses `!= null` at all other places...